pkg/resource/promtail-config/promtail-config.go

```diff
@@ -75,6 +76,11 @@ func GeneratePromtailConfig(lc loggedcluster.Interface) (v1.ConfigMap, error) {
 			},
 			Config: promtailConfig{
 				Snippets: promtailConfigSnippets{
+					ExtraRelabelConfigs: `# we only scrape logs from kube-system and giantswarm namespaces
```

Comment:
I think this should only apply to WCs though right?

Reply:
Hummm I totally forgot that point